.github/workflows/ci.yml

```
@@ -73,6 +78,93 @@ jobs:
           path: build/Amplitron
           retention-days: 1
 
+
+          coverage-linux:
```

actionlint (1.7.12)

[error] input "coverage-linux" is not defined in action "actions/upload-artifact@v7". available inputs are "archive", "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)